mix.exs

```diff
@@ -19,7 +19,7 @@ defmodule Glicko.Mixfile do
 
   defp deps do
     [
-      {:inch_ex, "~> 0.5", only: :docs},
+      {:inch_ex, "~> 2.0", only: :dev},
```

Comment:
This should probably be `[:dev, :docs]`, due to this [travis job](https://github.com/avitex/elixir-glicko/blob/master/.travis.yml#L14)